src/ts/fileDescriptorTSServices.ts

```diff
@@ -58,7 +58,7 @@ export function printFileDescriptorTSServices(fileDescriptor: FileDescriptorProt
      * from the given gRPC ${service.getName()} service.`));
       const requestMessageTypeName = getFieldType(MESSAGE_TYPE, method.getInputType().slice(1), "", exportMap);
       const responseMessageTypeName = getFieldType(MESSAGE_TYPE, method.getOutputType().slice(1), "", exportMap);
-      methodPrinter.printIndentedLn(`${lowercaseFirst(method.getName())}(request: ${requestMessageTypeName}): ${responseMessageTypeName};`);
+      methodPrinter.printIndentedLn(`async ${lowercaseFirst(method.getName())}(request: ${requestMessageTypeName}): Promise<${responseMessageTypeName}>;`);
```

Comment:
Interface should not annote the function as `async` only need to annote the return type as promise

Reply:
OK. I am going to remove that.